components/shared/Tailwind_Btn/TwButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Link from 'next/link'
+import { Navigation } from 'components/enum/navigation'
+
+
+interface TwButtonProps {
+    href: Navigation;
+    title: string;
+}
+const TwButton: React.FC<TwButtonProps> = ({ href, title }) => {
```

Comment:
Let's just call this `Button`

Reply:
changed function name, export, and interface.